src/Plugin/Hal.php

```diff
@@ -23,8 +24,9 @@
 use ZF\Hal\Collection;
 use ZF\Hal\Entity;
 use ZF\Hal\EntityHydratorManager;
-use ZF\Hal\Extractor\EntityExtractor;
 use ZF\Hal\Exception;
+use ZF\Hal\Extractor\EntityExtractor;
+use ZF\Hal\Extractor\LinkCollectionExtractor;
```

Comment:
You should rather update `linkCollectionExtractor` property annotation with `LinkCollectionExtractorInterface` instead of `LinkCollectionExtractor` and remove useless `LinkCollectionExtractor` alias.